scripts/ci/setup_env.sh

```diff
@@ -70,6 +70,7 @@ if [ $HADOOP_DISTRO = "cdh" ]; then
     HIVE_URL="http://archive.cloudera.com/cdh5/cdh/5/hive-latest.tar.gz"
 elif [ $HADOOP_DISTRO = "hdp" ]; then
     URL="http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.0.6.0/tars/hadoop-2.2.0.2.0.6.0-76.tar.gz"
+    HIVE_URL="http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.0.6.0/tars/hive-0.12.0.2.0.6.0-76.tar.gz"
```

Comment:
Can you please use a more recent hive? Hive 1.2.1 is stable in HDP Hive 2.0.0 soon. 0.12 is really old.